controllers/bentodeployment_controller.go

```diff
@@ -2255,7 +2255,11 @@ monitoring.options.insecure=true`
 		lastPort++
 		monitorExporterProbePort := lastPort
 
-		monitorExporterImage := "quay.io/bentoml/bentoml-monitor-exporter:0.0.2"
+		monitorExporterImage := "quay.io/bentoml/bentoml-monitor-exporter:0.0.3"
```

Comment:
How about renaming this value to `defaultMonitorExporterImage` and changing `monitorExporterImage_` to `monitorExporterImage`?